### PR TITLE
fix(i18n): update i18n

### PR DESCRIPTION
**Pull request by jare-talend**
**What is the problem this PR is trying to solve?**

The i18n context isn't taken in count in the component.

**What is the chosen solution to this problem?**

setI18n overrides the i18n instance. Now, we set i18n instance only if the host application hasn't one i18n config

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [x] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[X] This PR introduces a breaking change**

**Don't use the hoc getTranslated**

The getTranslated HOC decorates your component with one i18n provider. Now, The i18n provider is provided by your i18n config.

```diff
-import getTranslated from '@talend/react-components/lib/TranslateWrapper';

-	.directive('pureAppHeaderBar', ['reactDirective', reactDirective => reactDirective(
-		getTranslated(AppHeaderBar, { i18n })
-	)])
+       .directive('pureAppHeaderBar', ['reactDirective', reactDirective => reactDirective(AppHeaderBar)]);
```

### CODE REVIEW

File: packages/components/package.json [hunk 1]
```diff
@@ -103,7 +103,7 @@
     "react-a11y": "^0.3.3",
     "react-dnd-test-backend": "^2.5.4",
     "react-dom": "^16.8.6",
-    "react-i18next": "^10.10.0",
+    "react-i18next": "^10.11.4",
```
Comment (jmfrancois) [inline packages/components/package.json]: please udpate it in versions script

File: packages/datagrid/package.json [hunk 2]
```diff
@@ -73,7 +73,7 @@
     "prop-types": "^15.5.10",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-i18next": "^10.10.0",
+    "react-i18next": "^10.11.4",
```
Comment (jmfrancois) [inline packages/datagrid/package.json]: same here

Comment (jare-talend) [reply packages/datagrid/package.json]: done

File: packages/components/package.json (hunk 1 above)
Comment (jare-talend) [reply packages/components/package.json]: done

**Pull request merged**